gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py

```diff
@@ -288,6 +293,8 @@ def main():
     for test_case_configs in test_configs:
       test_cases = []
       for config in test_case_configs:
+        if args.test_name_prefix != '':
```

Comment:
nit: just `if args.test_name_prefix` is more pythonic :)

Reply:
done.